Source/common/SNTRule.m

```diff
@@ -28,6 +28,11 @@
 
 @interface SNTRule ()
 @property(readwrite) NSUInteger timestamp;
+@property(readwrite) SNTRuleState state;
+@property(readwrite) SNTRuleType type;
+@property(readwrite) NSString *customMsg;
+@property(readwrite) NSString *identifier;
+@property(readwrite) NSString *celExpr;
```

Comment:
Nit: Consider adding `comment` and `customURL` so our future selves don't hit a compiler issue